docs/development_suite/deploy/index.md

```diff
@@ -0,0 +1,33 @@
+---
+id: deploy
+title: Deploy your projects
+sidebar_label: Deploy your projects
```

Comment:
Not sure that this renaming is the right choice, since the "Design you projects" item is really similar and can cause confusion when using the sidebar